packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-java-visitor.ts

```diff
@@ -753,6 +761,55 @@ export class AppSyncModelJavaVisitor<
     return ['SuppressWarnings("all")', ...annotations].filter(annotation => annotation);
   }
 
+  protected generateAuthRules(authDirectives: AuthDirective[]): string {
+    const rules: string[] = [];
+    authDirectives.forEach(directive => {
+      directive.arguments?.rules.forEach(rule => {
+        const authRule = [];
+        if (rule.allow === AuthStrategy.groups) {
+          authRule.push('allow = AuthStrategy.GROUPS');
+          if (rule.groupClaim) {
+            authRule.push(`groupClaim = "${rule.groupClaim}"`);
+          }
+          if (rule.groups) {
+            authRule.push(`groups = { ${rule.groups?.map(group => `"${group}"`).join(', ')} }`);
+          } else {
+            authRule.push(`groupsField = "${rule.groupField}"`);
+          }
+        } else {
+          switch (rule.allow) {
+            case AuthStrategy.owner:
+              authRule.push('allow = AuthStrategy.OWNER');
+              break;
+            case AuthStrategy.private:
+              authRule.push('allow = AuthStrategy.PRIVATE');
+              break;
+            case AuthStrategy.public:
+              authRule.push('allow = AuthStrategy.PUBLIC');
+              break;
+            default:
+              printWarning(`Model has auth with authStrategy ${rule.allow} of which is not yet supported in DataStore.`);
```

Comment:
```suggestion
              printWarning(`Model has auth with authStrategy ${rule.allow} of which is not yet supported in Amplify.`);
```
The @auth directive isn't really specific to DataStore, it affects both the API and DataStore categories, so I would suggest saying this isn't supported in Amplify.

Reply:
Changed to "not yet supported"